### PR TITLE
Add 'Upgrading Hardened Clusters from v1.24.x to v1.25.x'  to page's table of contents

### PR DESCRIPTION
**Pull request by mantis-toboggan-md**
Per https://github.com/rancher/dashboard/issues/8442 we'd like to be able to link directly to this section in the dashboard.

### CODE REVIEW

**Comment by mantis-toboggan-md** [comment]
I don't have the ability to assign reviewers in this repo  but @brandond could you take a look at this?

**Review by brandond — APPROVED** [review]

File: docs/known_issues.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ spec:
 ```
 For more information, refer to comments on https://github.com/rancher/rke2/issues/3195.
 
-# Upgrading Hardened Clusters from v1.24.x to v1.25.x
+## Upgrading Hardened Clusters from v1.24.x to v1.25.x
```
Comment (brandond) [inline docs/known_issues.md:145]: We can probably make the anchor a bit shorter
```suggestion
## Upgrading Hardened Clusters from v1.24.x to v1.25.x {#hardened-125}
```

**Review by brandond — CHANGES_REQUESTED** [review]

**Review by brandond — APPROVED** [review]

File: docs/known_issues.md (hunk 1 above)
Comment (mantis-toboggan-md) [reply docs/known_issues.md]: Done

**Comment by mantis-toboggan-md** [comment]
Thank you for the ultra-fast response, Brad!

**Comment by brandond** [comment]
thanks for the PR. Should be live now!
https://docs.rke2.io/known_issues#hardened-125

**Pull request merged**